modules/lib/lib-auth/src/test/java/com/enonic/xp/lib/auth/LoginHandlerTest.java

```diff
@@ -75,6 +75,26 @@ public void testLoginSuccess()
         assertEquals( authInfo, sessionAuthInfo );
     }
 
+    @Test
+    public void testLoginWithScopeNONE()
+    {
+        ContextAccessor.current().getLocalScope().setSession( null );
+
+        final AuthenticationInfo authInfo =
+            AuthenticationInfo.create().user( TestDataFixtures.getTestUser() ).principals( RoleKeys.ADMIN_LOGIN ).build();
+
+        final IdProviders idProviders =
+            IdProviders.from( IdProvider.create().displayName( "system" ).key( IdProviderKey.from( "system" ) ).build() );
+
+        Mockito.when( this.securityService.authenticate( Mockito.any() ) ).thenReturn( authInfo );
+        Mockito.when( this.securityService.getIdProviders() ).thenReturn( idProviders );
+
+
+        runFunction( "/test/login-test.js", "loginWithScopeNONE" );
+
+        assertNull( ContextAccessor.current().getLocalScope().getSession() );
```

Comment:
should you also check that there is no AuthInfo attribute?

Reply:
Ah, sure. I thought that will be enough to check session, because that attribute is added to session if last exists